test/e2e/nsx_security_policy_test.go

```diff
@@ -185,37 +193,42 @@ func testSecurityPolicyMatchExpression(t *testing.T) {
 	securityPolicyName := "expression-policy-1"
 	ruleName := "expression-policy-1-rule"
 
-	var err error
-	setupTest(t, ns)
-	defer teardownTest(t, ns, defaultTimeout)
+	err := testData.createVCNamespace(ns)
+	if err != nil {
+		testData.deleteVCNamespace(ns)
```

Comment:
The testData.deleteVCNamespace(ns) is called both in the error-handling block and the defer statement?

Reply:
slip of the hand，fixed